src/lib/storage/chunk.hpp

```diff
@@ -24,7 +28,7 @@ namespace opossum {
 class BaseIndex;
 class BaseColumn;
 
-enum class ChunkUseAccessCounter { Yes, No };
+using ChunkColumns = pmr_concurrent_vector<std::shared_ptr<BaseColumn>>;
```

Comment:
Why does it have to be a concurrent vector if chunks are immutable anyway?

Reply:
Yeah, I guess than can become a std::vector now.

Reply:
Sadly, Chunk::migrate won't work with a normal std::vector

Reply:
Why shouldn’t it? You just have to change the type there too!

Reply:
oh yeah stupid me.

Reply:
wanted to write a sarcastic comment but this suggestion **actually works** if you also remove the PolymorphicAllocator from the ChunkColumns as well i Chunk::migrate(). But I'm not doing that in this PR, even though I'm in favour of this in general.

Reply:
Dude, just use `pmr_vector`!

Reply:
I don't even want to get into this allocator nonsense....